microsoft-edge/visual-studio/index.md

```diff
@@ -115,13 +112,13 @@ From the **Attach to Process** dialog, set **Connection type** to **Chrome devto
    Configure the **Attach to Process** dialog in Visual Studio  
 :::image-end:::  
 
-Choose **Select...** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, choose the **Refresh** button.  Choose the tab you want to debug and choose **Attach**.  
+Select **Select** > the checkbox next to **JavaScript (Microsoft Edge – Chromium)**.  To add tabs, navigate to new tabs, and close tabs and display the changes reflected in the **Attach to Process** dialog, select the **Refresh** button.  Choose the tab you want to debug and select **Attach**.  
 
 The Visual Studio debugger is now attached to Microsoft Edge.  You may pause the running of JavaScript, set breakpoints, and review `console.log()` statements directly in the Debug Output window in Visual Studio.  
```

Comment:
```suggestion
The Visual Studio debugger is now attached to Microsoft Edge.  You can pause the running of JavaScript, set breakpoints, and review `console.log()` statements directly in the Debug Output window in Visual Studio.  
```
I'd keep this left-justified.  Optionally shift w/ leading spaces to start in column 4(?) if it looks more consistent.